### PR TITLE
First draft of a new content plugins page

### PR DESCRIPTION
**Pull request by melcorr**
I don't think this page will work long-term but think it will work for now. 
It will serve to clarify the points causing confusion while I work on more robust pages.

The galaxyNG plugin is in the table but not in the Features tab because I discovered just as I added the last one that I had hit the limit. So this is a limitation I probably won't get around. I will kill this tabularised feature page once I write out individual pages. I still need to think about how I approach that. 

This is just a start, a short-term page. 

Any and all feedback welcome.

### CODE REVIEW

File: content-plugins.md [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+            <li class=""><a href="#deb" data-toggle="tab"><i class="fa fa-stop-circle"></i>Debian</a>
+            </li>
+            <li class=""><a href="#python" data-toggle="tab"><i class="fa fa-heart"></i> Python</a>
+            </li>
+            <li class=""><a href="#gem" data-toggle="tab"><i class="fa fa-play"></i> Gem</a>
+            </li>
+            <li class=""><a href="#cookbook" data-toggle="tab"><i class="fa fa-cutlery"></i> Cookbook</a>
+            </li>
+            <li class=""><a href="#maven" data-toggle="tab"><i class="fa fa-book"></i> Maven</a>
+            </li>
```
Comment (mdellweg) [inline content-plugins.md]: I see some inconsistent spacing with Container and debian here between the icon and the label.
Also the switching of the tables did not work for me. I do not immediately see whether its due to wrong classes or misconfigured js.

Comment (melcorr) [reply content-plugins.md]: > I see some inconsistent spacing with Container and debian here between the icon and the label.

Ah yeah. I had run into space problems, and was trying to troubleshoot, and forgot to change back. Thanks. Will fix. 

>Also the switching of the tables did not work for me. I do not immediately see whether its due to wrong classes or misconfigured js.

Really? This works for me both on the local and github.io site. What browser are you using?

Comment (melcorr) [reply content-plugins.md]: I've fixed the spacing issue but have checked on my mobile and gotten my pal to check on his browser and cannot reproduce what you're seeing with the other issue.

File: content-plugins.md [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+toc: true
+---
+
+## Content Plugins
+
+After you install Pulp, you must add a content plugin for each content type that you want to manage. The following sections contain information about the available content plugins for both Pulp 2 and Pulp 3. If you do not find the plugin for the content type you want to manage, consider [writing a plugin](https://docs.pulpproject.org/plugins/plugin-writer/index.html).
+
+## Plugin Changes between Pulp 2 and Pulp 3
+
+* The ISO functionality found in the Pulp 2 RPM plugin has been replaced by the File plugin.
```
Comment (RCMariko) [inline content-plugins.md]: Suggest:
- "ISO content support" (does that sound better?)
-  "replaced" --> "is now provided" - or just say was moved to it's own File plugin.

**Comment by RCMariko** [comment]
I'm excited about these changes! Looks fantastic!

**Comment by RCMariko** [comment]
A thousand pardons....hit the wrong button (close pr). If I haven't fixed my mistake, please let m know.

File: content-plugins.md (hunk 2 above)
Comment (melcorr) [reply content-plugins.md]: Excellent. Thank you for this. I have applied your feedback.

File: content-plugins.md [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+    </div>
+</div>
+
+## Pulp 3 Content Plugins Information
+
+This table contains links to information and sources for all Pulp 3 content plugins. If a plugin is missing [contact us](https://www.redhat.com/mailman/listinfo/pulp-list).
+
+| Pulp Plugin | Docs | Source | Tracker | Install with PyPI | Install with RPM |
+|-------|--------|---------|--------|---------|-------- |--------- |
+| Ansible plugin | <a href="https://github.com/pulp/pulp_ansible/blob/master/README.rst">Ansible plugin docs</a> | <a href="https://github.com/pulp/pulp_ansible">Ansible plugin source</a> | <a href="https://pulp.plan.io/projects/ansible_plugin?jump=welcome">Ansible plugin tracker</a> | <a href="https://pypi.org/project/pulp-ansible/">Yes</a> | No |
```
Comment (melcorr) [inline content-plugins.md]: Note to self: replace this doc link with: https://pulp-ansible.readthedocs.io/en/latest/

File: content-plugins.md [hunk 4]
```diff
@@ -0,0 +1,245 @@
...
+| Python plugin | <a href="http://pulp-python.readthedocs.io/en/latest/">Python plugin docs</a> | <a href="https://github.com/pulp/pulp_python/">Python plugin source</a> | <a href="https://pulp.plan.io/projects/pulp_python?jump=welcome">Python plugin tracker</a> | <a href="https://pypi.org/project/pulp-python/">Yes</a> | No |
+| RPM plugin | <a href="http://pulp-rpm.readthedocs.io/en/latest/">RPM plugin docs</a> | <a href="https://github.com/pulp/pulp_rpm/">RPM plugin source</a> | <a href="https://pulp.plan.io/projects/pulp_rpm?jump=welcome">RPM plugin tracker</a> | <a href="https://pypi.org/project/pulp-rpm/">Yes</a> | No |
+
+## Pulp 3 Extras
+
+| Pulp Plugin | Docs | Source | Tracker | Install with PyPI | Install with RPM |
+|-------|--------|---------|--------|---------|-------- |--------- |
+| pulp-certguard | <a href="https://github.com/pulp/pulp-certguard/blob/master/README.rst">Certguard Docs</a> | <a href="https://github.com/pulp/pulp-certguard">Certguard source</a> | <a href="https://pulp.plan.io/projects/certguard?jump=welcome">Certguard plugin tracker</a> | <a href="https://pypi.org/project/pulp-certguard/">Yes</a> | No |
+
```
Comment (melcorr) [inline content-plugins.md:157]: Replace this one here: https://pulp-certguard.readthedocs.io/en/latest/

File: content-plugins.md (hunk 1 above)
Comment (mdellweg) [reply content-plugins.md]: Ok, today it just works. So please disregard my comment about that.

**Comment by mdellweg** [comment]
Is GalaxyNG a content plugin (ping @bmbouter)?
Maybe it belongs to the extra section.

File: content-plugins.md (hunk 1 above)
Comment (melcorr) [reply content-plugins.md]: Thanks!

**Comment by bmbouter** [comment]
> Is GalaxyNG a content plugin (ping @bmbouter)?
> Maybe it belongs to the extra section.

It does add a UI for content plugins. That kind of puts it in a grey area. I'm ok with it being in either area.

**Comment by melcorr** [comment]
> > Is GalaxyNG a content plugin (ping @bmbouter)?
> > Maybe it belongs to the extra section.
> 
> That kind of puts it in a grey area. 

@bmbouter , from the GalaxyNG docs, it is hard to make the distinction between that and the Ansible plugin, could you provide some information here to distinguish them?

**Pull request merged**